api/src/pcapi/routes/backoffice/offer_price_limitation_rules/forms.py

```diff
@@ -0,0 +1,49 @@
+from flask_wtf import FlaskForm
+import wtforms
+
+from pcapi.core.categories import categories
+from pcapi.core.categories import subcategories_v2
+
+from ..forms import fields
+from ..forms import utils
+
+
+class SearchRuleForm(FlaskForm):
+    class Meta:
+        csrf = False
+
+    category = fields.PCSelectMultipleField(
+        "Catégories", choices=utils.choices_from_enum(categories.CategoryIdLabelEnum)
+    )
+    subcategory = fields.PCSelectMultipleField(
+        "Sous-catégories",
+        choices=sorted(
+            [(s.id, s.pro_label) for s in subcategories_v2.ALL_SUBCATEGORIES if not s.is_event], key=lambda x: x[1]
+        ),
+    )
+
+
+class EditOfferPriceLimitationRuleForm(FlaskForm):
+    class Meta:
+        locales = ["fr_FR", "fr"]
+
+    rate = fields.PCDecimalField(
+        "Limite de modification du prix %",
+        use_locale=True,
+        validators=[
+            wtforms.validators.InputRequired("Information obligatoire"),
+            wtforms.validators.NumberRange(min=0, message="Doit contenir un nombre positif"),
+        ],
+    )
+
+
+class CreateOfferPriceLimitationRuleForm(EditOfferPriceLimitationRuleForm):
+    class Meta:
+        locales = ["fr_FR", "fr"]
+
+    subcategory = fields.PCSelectField(
+        "Sous-catégorie",
+        choices=sorted(
+            [(s.id, s.pro_label) for s in subcategories_v2.ALL_SUBCATEGORIES if not s.is_event], key=lambda x: x[1]
+        ),
+    )
```

Comment:
C'est dommage d'avoir la sous-catégorie après le pourcentage : 
```suggestion
    )
    
    def __init__(self, *args: typing.Any, **kwargs: typing.Any) -> None:
        super().__init__(*args, **kwargs)
        self._fields.move_to_end("subcategory", last=False)
```